_stbt/core.py

```diff
@@ -511,17 +511,20 @@ class MotionResult(object):
     * ``motion``: Boolean result, the same as evaluating `MotionResult` as a
       bool. That is, ``if result:`` will behave the same as
       ``if result.motion:``.
+    * ``region``: The region of the video frame that contained the motion.
+      `None` if no motion detected.
     """
-    def __init__(self, timestamp, motion):
+    def __init__(self, timestamp, motion, region):
         self.timestamp = timestamp
         self.motion = motion
+        self.region = region
```

Comment:
In the past we've made new parameters to these constructors optional, for backwards compatibility. See the `MatchResult` constructor.  I can imagine someone having written a `wait_for_no_motion` helper function that constructs their own `MotionResult` object. What do you think, was this a conscious decision?